lib/pbench/server/api/resources/datasets_inventory.py

```diff
@@ -84,10 +85,9 @@ def _get(
         # We link a callback to close the file stream and TarFile object on
         # completion.
```

Comment:
FWIW, there no longer is a "TarFile object" associated with the stream....

Reply:
Yeah...if only it had occurred to him that code which didn't reference the object in code might have a reference to it in a docstring....